apps/token/src/hooks/use-erc-poll20-approval.ts

```diff
@@ -2,13 +2,13 @@ import React from 'react';
 import { gql, useApolloClient } from '@apollo/client';
 
 import type {
-  Erc20Approval,
-  Erc20Approval_erc20WithdrawalApproval,
-  Erc20ApprovalVariables,
-} from './__generated__/Erc20Approval';
+  Erc20ApprovalPoll,
+  Erc20ApprovalPoll_erc20WithdrawalApproval,
+  Erc20ApprovalPollVariables,
+} from './__generated__/Erc20ApprovalPoll';
 
 const ERC20_APPROVAL_QUERY = gql`
-  query Erc20Approval($withdrawalId: ID!) {
+  query Erc20ApprovalPoll($withdrawalId: ID!) {
```

Comment:
**thought (non-blocking)**: By putting 'Poll' in the query name you made me realise that (assuming we're still using Apollo link splitting) you *could* use the query name to ensure that polling queries go on the websocket rather than HTTP, while leaving other queries on HTTP, just by checking that the query name ends with 'Poll'. It would only be a minor benefit - mainly tidying up devtools - but it would be marginally more efficient network request wise.

Reply:
Thats a nice idea, hopefully we won't need as much polling going forward, but if we do this would be handy